src/main/charts/bitbucket/templates/tests/test-shared-home-permissions.yaml

```diff
@@ -17,8 +17,8 @@ spec:
       imagePullPolicy: IfNotPresent
       securityContext:
         # Slightly dodgy; we assume that the UID and GID used by the product images are the same, which in practice they are
-        runAsUser: {{ .Values.bitbucket.securityContext.gid }}
-        runAsGroup: {{ .Values.bitbucket.securityContext.gid }}
+        runAsUser: {{ .Values.bitbucket.securityContext.fsGroup }}
```

Comment:
What would happen if somebody overrides the full security context and leave out the `fsGroup`? When we say that users have full flexibility to change the `securityContext`, they can probably also include the context without `fsGroup. We should either cater for the default or document that.